packages/clerk-js/src/ui/contexts/ClerkUIComponentsContext.tsx

```diff
@@ -310,11 +313,17 @@ export const useOrganizationSwitcherContext = () => {
 
   const navigateAfterSelectPersonal = (user: UserResource) => navigateAfterSelectOrganizationOrPersonal({ user });
 
+  const organizationProfileMode =
+    !!ctx.organizationProfileUrl && !ctx.organizationProfileMode ? 'navigation' : ctx.organizationProfileMode;
+
+  const createOrganizationMode =
+    !!ctx.createOrganizationUrl && !ctx.createOrganizationMode ? 'navigation' : ctx.createOrganizationMode;
+
```

Comment:
Quick question, do we have similar helpers for the other components? If no, what is the reason there are needed here?

Reply:
Yeah it's similar to [normalizeRoutingOptions](https://github.com/clerk/javascript/pull/1957/files/c4a994eb78c67d5fbae58badc3d03372cad3b42e#diff-f7fa0cdb70cc4f092d8ae5107fa3aacbc3bf2917ad56442ab5dd98169705d20eR116), this could be extracted as a helper too.